app/livechat/server/hooks/markRoomNotResponded.ts

```diff
@@ -15,11 +21,11 @@ callbacks.add(
 		}
 
 		// check if room is yet awaiting for response
-		if (typeof room.t !== 'undefined' && room.t === 'l' && room.waitingResponse) {
+		if (room.t === 'l' && room.isWaitingResponse) {
 			return message;
 		}
 
-		LivechatRooms.setNotResponseByRoomId(room._id);
+		LivechatRooms.setNotResponseByRoomId(room?._id);
```

Comment:
```suggestion
		LivechatRooms.setNotResponseByRoomId(room._id);
```

Room should be defined after L24 🤔 can you try this?